retro/retro_env.py

```diff
@@ -191,8 +206,10 @@ def step(self, a):
         if self.movie:
             self.movie.step()
         self.em.step()
-        self.img = ob = self.get_screen()
+        self.img = self.em.get_screen()
```

Comment:
Crops are intentional.

Reply:
My bad, I thought it was inconsistent about crops, but I must have mis-read the code.